### PR TITLE
gdb fixes (#11087, #11088)

### PR DESCRIPTION
**Pull request by AndWac**

### CODE REVIEW

File: components/developer/gdb/Makefile [hunk 1]
```diff
@@ -20,24 +20,25 @@
...
+# Copyright 2019 Andreas Wacknitz
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		gdb
 COMPONENT_VERSION=	7.10.1
-COMPONENT_REVISION=	4
+COMPONENT_REVISION=	5
+COMPONENT_SUMMARY=  GDB: The GNU Project Debugger
```
Comment (Mno-hime) [inline components/developer/gdb/Makefile]: Tab might do better.

**Review by Mno-hime — APPROVED** [review]

File: components/developer/gdb/Makefile (hunk 1 above)
Comment (AndWac) [reply components/developer/gdb/Makefile]: I have tried to fix the tabs, does it look better now?

**Comment by pyhalov** [comment]
Regenerated syscall patches with something like 
```
grep 'define     SYS' /usr/include/sys/syscall.h  |sed -e 's:SYS_::' | awk ' { print "  <syscall name=\"" $2 "\" number=\"" $3"\"/>" ; }'
``` 
and manual fixing.

**Review by richlowe — APPROVED** [review]
That definitely looks like our syscall table now!

**Comment by richlowe** [comment]
This still leaves a (largely?) non-functional gdb.  Perhaps everything in the patch that introduced this needs regenerating?

I notice that Oracle's gdb has fewer patches (though obviously they'd need regenerating for illumos too).

I also notice that pkgsrc doesn't patch gdb really at all.

**Pull request merged**